web/controllers/game_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Yggdrasil.GameController do
+  use Yggdrasil.Web, :controller
+
+  alias Yggdrasil.Game
+
+  def index(conn, _params) do
+    games = Repo.all Game
```

Comment:
Yeah, more of a fan of a suite of functions that do one thing well that can be composed than overly specific functions where it's like oh you need to call this one for that action or this other for that action and they may be disjoint where you need something in between.

A user for example I would rather the web completely own and have direct repo access, not that I'm shy or subtle about how I want to throw that back into the web :wink:.

I do have hard time thinking of any one example where you would be doing straight CUD (doesn't read very well without the R...haha) on any game related table though that the game isn't better suited in doing.

My opposition to direct access to the Game's repo is that it allows unchecked abilities to remove or update records that the web will most likely never know if it can safely do. That said I do see where the query abilities would be needed definitely.

That said, it's not the repo that offers the querying, it's the Ecto.Query module, so there is no reason the APIs can't take in a ecto query to execute and return results. I'm not opposed to exposing the ecto schemas for such things or use else where. 

Even still I have hard time picturing any game related tables that will be as simple as getting only data from the tables. Like if you want an admin to see characters in a game, but you want to allow the admin to say show me only characters actively playing the game. That is no longer just a db question, and maybe the it's a Character.query() function at that point where you give ecto query and it augments the results coming back to include an active flag which the UI can use to filter for example or can filter before the UI.

I was sort taking the stance that it's easier to slowly open up and expose access to underlying things than start wide open and trying to reign it back in. Like any case where what we provide doesn't satisfy a certain need we can step back and say ok what combo is needed what is the impact of that, and we can test it out isolated instead of through controllers etc. Then we end up with the bare minimum api we need as we go.

All that said, I'm ok if you prefer to start with direct repo access and iteratively pull it back into yggdrasil as well. I see your points and concerns as well.